.github/workflows/controller.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright Confidential Containers Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# Run tests for peerpod controller.
+---
+name: controller
+on: [pull_request]
+jobs:
+  controller_job:
+    name: controller
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: peer-pod-controller
+    strategy:
+      matrix:
+        go_version:
+          - 1.18
```

Comment:
Currently golang version is 1.18, will need update once this PR is merged - #648